src/gsi/gsi_rfv3io_mod.f90

```diff
@@ -1101,6 +1116,31 @@ subroutine read_fv3_netcdf_guess(fv3filenamegin)
 
       endif !laeroana_fv3cmaq
 
+      if (laeroana_fv3smoke) then
+        jtracer = 0
+!name_chemvars3d chemguess from anainfo
+        do i=1,size(name_chemvars3d)
+          vartem=trim(name_chemvars3d(i))
+           if (ifindstrloc(aeronames_smoke_fv3,trim(vartem)) > 0) then
+             jtracer=jtracer+1
+             fv3lam_io_tracersmokevars3d_nouv(jtracer)=trim(vartem)
+              write(6,*)'the chemvarname ',jtracer,vartem,' is found '
+           else
+              write(6,*)'the chemvarname ',vartem,' is not in aeronames_smoke_fv3, !!!!!!!!!!'
+              !call flush(6)
+              !call stop2(333)
+           endif
+        enddo
+        ntracersmokeio3d=jtracer+1
+        fv3lam_io_tracersmokevars3d_nouv(jtracer+1)="pm2_5"
```

Comment:
probably a dumb question, but what does `nouv` mean on all of these? is it 'no wind' or 'no ultraviolet'? or something else?